setup.py

```diff
@@ -127,5 +127,10 @@ def read(fname):
     download_url='%s/v%s.tar.gz' % (url, version),
     keywords=['OMERO.CLI', 'plugin'],
     cmdclass={'test': PyTest},
-    tests_require=['pytest'],
+    install_requires=[
+        'omero-py>=5.6.dev4',
+        'future'],
+    tests_require=[
+        'pytest',
+        'mox3'],
```

Comment:
See https://travis-ci.org/sbesson/omero-upload/builds/613717629 which attempts to remove the `mox3` dependency. `omero.testlib` has a dependency on `mox3` but the latter is only declared in the upstream `tests_require. This might be a good use case for `extra_requires`. I'll open a PR to start the discussion

Reply:
As discussed let's leave this in for now